persistence/src/main/java/de/cyface/persistence/PersistenceLayer.java

```diff
@@ -699,24 +703,41 @@ public Double loadAscend(final long measurementIdentifier, final boolean forceGn
     Double ascendFromPressures(final List<TrackV6> tracks) {
         Double totalAscend = null;
         for (final TrackV6 track : tracks) {
-            Double ascend = null;
-            Double lastAltitude = null;
-            for (final Pressure pressure : track.getPressures()) {
 
+            // Calculate average pressure because some devices measure large pressure differences when
+            // the display-fingerprint is used and pressure is applied to the display: Pixel 6 [STAD-400]
+            // This filter did not affect ascend calculation of devices without the bug: Pixel 3a
```

Comment:
See STAD-400

Without this fix, the ascend on the Pixel 6 would be
- 1-6 km instead of 90 m, while on the Pixel 3a it's 87 m.
- 200-600 m instead of 0-5 m, while on the Pixel 3a it's 0 m

With the two average filters applied, both return 87 m and ~ 0-5 m.